tests/keras/backend/backend_test.py

```diff
@@ -1802,6 +1802,22 @@ def test_sparse_concat(self):
         assert k_s_d.shape == k_d.shape
         assert_allclose(k_s_d, k_d, atol=1e-05)
 
+    def test_stack(self):
+        tensor_list = [np.random.randn(5, 4, 6, 10) for _ in range(5)]
+        stack_axis = 3
+        results = []
+        for k in WITH_NP:
+            if k == KC:
+                check_two_tensor_operation('stack', (5, 4, 6, 10),
+                                           (5, 4, 6, 10), WITH_NP,
+                                           axis=stack_axis, concat_args=True)
+            else:
+                tensor_list_var = [k.variable(tensor) for tensor in tensor_list]
+                out = k.eval(k.stack(tensor_list_var, axis=stack_axis))
+                results.append(out)
+
+        assert_list_pairwise(results)
```

Comment:
Just noticed that this line doesnt test anything if backend is CNTK. So one last nit pick.. Put the backend check outside the loop so that it makes more sense:

```python
    def test_stack(self):
        tensor_list = [np.random.randn(5, 4, 6, 10) for _ in range(5)]
        stack_axis = 3
        results = []
        if K.backend() == 'cntk':
            check_two_tensor_operation('stack', (5, 4, 6, 10),
                                        (5, 4, 6, 10), WITH_NP,
                                        axis=stack_axis, concat_args=True)
        else:
            for k in WITH_NP:
                tensor_list_var = [k.variable(tensor) for tensor in tensor_list]
                out = k.eval(k.stack(tensor_list_var, axis=stack_axis))
                results.append(out)
             assert_list_pairwise(results)

```

Reply:
ok. sure.

Reply:
wait - I have an issue with K.backend() == 'cntk'. TF backend doesn't have a backend() function.

Reply:
I'll break the forloop if the first k in WITH_NP is cntk